src/internal/connector/sharepoint/data_collections.go

```diff
@@ -71,17 +80,53 @@ func DataCollections(
 				if err != nil {
 					return nil, support.WrapAndAppend(site, err, errs)
 				}
-
-				collections = append(collections, spcs...)
 			}
 
+			collections = append(collections, spcs...)
+
 			foldersComplete <- struct{}{}
 		}
 	}
 
 	return collections, errs
 }
 
+func collectLists(
+	ctx context.Context,
+	serv graph.Servicer,
+	tenantID, siteID string,
+	scope selectors.SharePointScope,
+	updater statusUpdater,
+	ctrlOpts control.Options,
+) ([]data.Collection, error) {
+	logger.Ctx(ctx).With("site", siteID).Debug("Creating SharePoint List Collections")
+
+	if scope.Matches(selectors.SharePointSite, siteID) {
+		spcs := make([]data.Collection, 0)
+
+		dir, err := path.Builder{}.Append(siteID).
```

Comment:
The site shouldn't be used as the folder.  If there isn't a list id/name that could be used here, we should at least use a static reference for "list".  Probably the path category string?

Reply:
Resolved with [9b3af77](https://github.com/alcionai/corso/pull/1788/commits/9b3af77c07de84819f531f1425078cd5b5a89e3b)